packages/rolldown/tests/fixtures/plugin/banner-with-str/_config.ts

```diff
@@ -0,0 +1,20 @@
+import { expect, vi } from 'vitest'
+import path from 'node:path'
+import { defineTest } from '@tests'
+
+const entry = path.join(__dirname, './main.js')
+
+export default defineTest({
+  config: {
+    input: entry,
+    plugins: [
+      {
+        name: 'test-plugin',
+        banner: '/* Lorem ipsum */',
```

Comment:
Could you add a test for this?

```
{

     banner: {
           handle: '// banner'
     }
}
```

Reply:
If you update  `/* Lorem ipsum */` to `/* banner */` will make output clear to see.